namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-preprod/02-limitrange.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limitrange
+  namespace: sentence-planning-preprod
+spec:
+  limits:
+  - default:
+      cpu: 250m
```

Comment:
Please increase the default cpu limit to 1000m - this is to allow enough capacity for any startup processing to complete quickly.

Reply:
I have made this change to our dev and preprop environments